src/test/java/edu/hm/hafner/analysis/parser/CodeGeneratorParserTest.java

```diff
@@ -0,0 +1,44 @@
+package edu.hm.hafner.analysis.parser;
+
+import edu.hm.hafner.analysis.AbstractParserTest;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.assertions.SoftAssertions;
+
+/**
+ * Tests the class {@link CodeGeneratorParser}.
+ *
+ *  @author Eva Habeeb
+ */
+class CodeGeneratorParserTest extends AbstractParserTest {
+    CodeGeneratorParserTest() {
+        super("CodeGenerator.log");
+    }
+
+    @Override
+    protected CodeGeneratorParser createParser() {
+        return new CodeGeneratorParser();
+    }
+
+    @Override
+    protected void assertThatIssuesArePresent(final Report report, final SoftAssertions softly) {
+        softly.assertThat(report).hasSize(8);
+        softly.assertThat(report.get(0))
+                .hasCategory("Configuration Parameters Unavailable")
+                .hasMessage("'A_123 interface' is no longer available in the Configuration Parameters dialog box and will be removed in a future release. Set 'GenerateASAP2' to 'off' by using thecommand line. To generate ASAP2 files, use the <a href=\"matlab:helpview(fullfile(docroot,'rtw','helptargets.map'),'export_asap2_cdf')\">\"Generate Calibration Files\"</a> tool instead.");
```

Comment:
Note that all HTML tags will be sanitized in Jenkins before shown in the UI.

Reply:
That should be fine.